packages/template-block/src/components/TemplateText.tsx

```diff
@@ -80,11 +80,8 @@ export const TemplateText = ({
             <div className="tw-mb-2">
```

Comment:
In this also make the id of the rte's unique with adding the blockId `appBridge.context('blockId').get()`

here it already is a problem if you have multiple template blocks per page

How to test:
- add 2 local dev blocks
- try to set a link via the rte

Reply:
Awesome, thanks for the hint - I had no idea! Works as expected now. 🚀 
![image](https://github.com/Frontify/guideline-blocks/assets/9266535/70dc3ae4-c020-409d-a62d-d0f7673d5ec1)